spec/index.bs

```diff
@@ -7334,6 +7334,17 @@ typedef double GPUPipelineConstantValue; // May represent WGSL's bool, f32, i32,
         </div>
 </dl>
 
+<div algorithm>
+    To get the <dfn abstract-op>entry point</dfn>({{GPUShaderStage}} |stage|,
+    {{GPUProgrammableStage}} |descriptor|)
+
+    - If |descriptor|.{{GPUProgrammableStage/entryPoint}} is not `null`, return
+        |descriptor|.{{GPUProgrammableStage/entryPoint}}.
+    - Return the name of the entry point in |descriptor|.{{GPUProgrammableStage/module}} for shader
+        stage |stage|.
```

Comment:
This is a bit handwavy but we want to return the entry point itself, not its name.
```suggestion
    - If |descriptor|.{{GPUProgrammableStage/entryPoint}} is not `null`:

        - Return the single entry point in |descriptor|.{{GPUProgrammableStage/module}}
            with a name equalling |descriptor|.{{GPUProgrammableStage/entryPoint}}.

        Otherwise:

        - Return the single entry point in |descriptor|.{{GPUProgrammableStage/module}}
            with a shader stage equalling |stage|.
```